mitosheet/mitosheet/step_performers/graph_steps/plotly_express_graphs.py

```diff
@@ -189,27 +195,65 @@ def graph_creation_code(
 
 
 def graph_styling(
-    fig: go.Figure, graph_type: str, column_headers: List[ColumnHeader], filtered: bool
+    fig: go.Figure, graph_type: str, column_headers: List[ColumnHeader], filtered: bool, graph_styling_params: Dict[str, Any]
 ) -> go.Figure:
     """
     Styles the Plotly express graph figure
     """
 
     # Create the parameters that we use to style the graph
-    graph_title = get_graph_title(column_headers, [], filtered, graph_type)
+    if graph_styling_params['title']['visible']:
+        graph_title = graph_styling_params['title']['title'] if 'title' in graph_styling_params['title'].keys() else get_graph_title(column_headers, [], filtered, graph_type)
```

Comment:
You can just say `if 'title' in graph_styling_params['title']`, you don't need to do the explicit .keys() call.